internal/controller/utils.go

```diff
@@ -432,6 +433,28 @@ func SetApisixRouteConditionAccepted(status *apiv2.ApisixStatus, generation int6
 	status.Conditions = []metav1.Condition{condition}
 }
 
+func SetApisixPluginConfigConditionAccepted(status *apiv2.ApisixStatus, generation int64, err error) {
```

Comment:
这个函数似乎对所有 apiv2.Apisix* 的资源都通用。
```suggestion
func SetApisixV2ConditionAccepted(status *apiv2.ApisixStatus, generation int64, err error) {
```